src/DDTrace/Integrations/Filesystem/FilesystemIntegration.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+namespace DDTrace\Integrations\Filesystem;
+
+use DDTrace\HookData;
+use DDTrace\Integrations\Integration;
+use DDTrace\Tag;
+
+class FilesystemIntegration extends Integration
+{
+    const NAME = "filesystem";
+
+    public function init(): int
+    {
+        if (!\dd_trace_env_config("DD_APPSEC_RASP_ENABLED")) {
+            return Integration::LOADED;
+        }
+
+        if (!function_exists('\datadog\appsec\push_address')) {
```

Comment:
```suggestion
        if (!function_exists('datadog\appsec\push_address')) {
```
Nit: the FQN is really without leading slash. Just some places in PHP allow it and strip it away...